### PR TITLE
adjust some syntax, remove error while charging

### PR DESCRIPTION
**Pull request by noahbei**
some minor indentations fixes

### CODE REVIEW

File: UI.js [hunk 1]
```diff
@@ -55,7 +55,8 @@ draw(context){
         if (this.game[type] === 6) context.drawImage(images.bars, 51, yDist, 6, 18, 65 + barX, -14, 18, 60);
     }
     drawBar("health");
-    drawBar("energy");    
+    drawBar("energy");
+
```
Comment (JeremiahHerring) [inline UI.js:59]: wow amazing!

File: capyStates.js [hunk 2]
```diff
@@ -126,7 +126,6 @@ export class Charging extends State {  // Child Class (sub class)
         // While a game.capy is in a certain state, it will only react to a certain amount of inputs
         if (!input.includes(' ') && this.game.capy.onGround()){
             this.game.capy.setState(states.WALKING, 1);
-            this.game.capy.y = 427.5;
```
Comment (JeremiahHerring) [inline capyStates.js:129]: nice!

**Review by JeremiahHerring — COMMENTED** [review]
nice work

File: capyStates.js (hunk 2 above)
Comment (noahbei) [reply capyStates.js:129]: good catch 👍

**Pull request merged**